GetARock/GetARock/Screen/MainMap/MainMapViewController.swift

```diff
@@ -108,3 +131,19 @@ extension MainMapViewController: CLLocationManagerDelegate {
     }
     
 }
+
+// MARK: - CLLocationManagerDelegate
+
+extension MainMapViewController: MKMapViewDelegate {
+    func mapView(_ mapView: MKMapView, viewFor annotation: MKAnnotation) -> MKAnnotationView? {
+        if annotation as? MKUserLocation != nil {
+            return MKUserLocationView()
+        }
```

Comment:
```suggestion
        if annotation is? MKUserLocation {
            return MKUserLocationView()
        }
```
p3;
타입체크를 하려는 거면 가독성을 위해 is 키워드를 사용하는 것이 어떨까요?

Reply:
현재 PR이 꼬여서 다른 브랜치에서 수정된 사항인데 여기서도 변경해둬야 할까요??